app/controllers/photos_controller.rb

```diff
@@ -0,0 +1,37 @@
+class PhotosController < ApplicationController
+  include InheritAction
+
+  def create
+    @album = Album.find(params[:album_id])
+    @photos = @album.photos.create(photo_params)
+    render_success_response({ :photos => @photos}, 201)
+  end
+
+  def multi_destroy
+    @album = Album.find(params[:album_id])
```

Comment:
@arps1989dev  Please refactor the code of `multi_destroy` method and rename the method name to `multi_delete` instead of `multi_destroy`.